### ISSUE TITLE
Inconsistent Go Vendor

### ISSUE THREAD

**Issue by krishvoor** [danielqsj/kafka_exporter#208]
With Go-1.16 crossbuild, we need to revisit the go-vendor 

```root:/home/gopath/src/src/github.com/danielqsj/kafka_exporter# make
>> formatting code
go: downloading golang.org/x/sys v0.0.0-20210315160823-c6e025ad8005
go: downloading golang.org/x/net v0.0.0-20210315170653-34ac3e1c2000
go get: upgraded github.com/prometheus/promu v0.8.1 => v0.9.0
go get: upgraded golang.org/x/net v0.0.0-20210315170653-34ac3e1c2000 => v0.0.0-20210316092652-d523dce5a7f4
go get: upgraded golang.org/x/sys v0.0.0-20210315160823-c6e025ad8005 => v0.0.0-20210317091845-390168757d9c
>> building binaries
 >   kafka_exporter
go: inconsistent vendoring in /home/gopath/src/src/github.com/danielqsj/kafka_exporter:
	github.com/prometheus/promu@v0.9.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	golang.org/x/net@v0.0.0-20210316092652-d523dce5a7f4: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	golang.org/x/sys@v0.0.0-20210317091845-390168757d9c: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
	golang.org/x/net@v0.0.0-20210315170653-34ac3e1c2000: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod
	golang.org/x/sys@v0.0.0-20210315160823-c6e025ad8005: is marked as explicit in vendor/modules.txt, but not explicitly required in go.mod

	To ignore the vendor directory, use -mod=readonly or -mod=mod.
	To sync the vendor directory, run:
		go mod vendor
!! command failed: build -o /home/gopath/src/src/github.com/danielqsj/kafka_exporter/kafka_exporter -ldflags -X github.com/danielqsj/kafka_exporter/vendor/github.com/prometheus/common/version.Version=1.2.0 -X github.com/danielqsj/kafka_exporter/vendor/github.com/prometheus/common/version.Revision=d7f1a13b72916c1321c1fb04294f336448facf4a -X github.com/danielqsj/kafka_exporter/vendor/github.com/prometheus/common/version.Branch=master -X github.com/danielqsj/kafka_exporter/vendor/github.com/prometheus/common/version.BuildUser=root@p006vm81 -X github.com/danielqsj/kafka_exporter/vendor/github.com/prometheus/common/version.BuildDate=20210317-13:57:13  -extldflags '-static' -a -tags netgo github.com/danielqsj/kafka_exporter: exit status 1
Makefile:30: recipe for target 'build' failed
make: *** [build] Error 1
root:/home/gopath/src/src/github.com/danielqsj/kafka_exporter# 
```

**Comment by krishvoor** [comment]
I have the PR, validating this across x86_64 and ppc64le architecture, will the push the PR soon.

**Comment by krishvoor** [comment]
@danielqsj  - Please review this - 
https://github.com/danielqsj/kafka_exporter/pull/209

**Comment by krishvoor** [comment]
Looks like, when I make use of Go-1.16.x I see the Go vendoring issue, however, with Go-1.14 I do not see the issue.
Do we Go-1.16.X , for the latest master branch?

**Comment by caorong** [comment]
you can just remove vendor 

```
mv vendor _vendor
```

> Looks like, when I make use of Go-1.16.x I see the Go vendoring issue, however, with Go-1.14 I do not see the issue.
> Do we Go-1.16.X , for the latest master branch?

**Comment by krishvoor** [comment]
This "temporary fix" will let us steer ahead, this job that would fail in a CI/CD with Go-16.x because of inconsistent go.mod/ go.sum, as documented earlier, when executed with the make command.

> you can just remove vendor

Since currently no artefacts in form of the binaries are published, it becomes further important to update,
if not document this behaviour.

**Comment by danielqsj** [comment]
Hi all, I fix this issue, and released 1.3.1. :)
